README.md

```diff
@@ -14,18 +14,63 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+## How-to-install
+### Instal for Ubuntu based distro (Ubuntu 16.04)
+It is recommended to install packages with APT (change stable to `master` or `rc` if needed):
+    
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial stable"
+    sudo apt-get update
+    sudo apt-get install -y libindy
+    
+### Instal for Windows
+1. follow to https://repo.sovrin.org/windows/libindy.
+2. download last version of libindy and indy-sdk-deps.
+3. unzip archives to directories, where you want to save working libraries.
+4. add to PATH enviroment variable path to that directories.
+
+### Instal for MacOS        
+Now we haven't prebuild library in some shared place. You can build
+library yourself. Please refer to How-to-build section. 
+After build add to PATH enviroment variable path to builded library.
```

Comment:
Please re-check name of the environment variable. Most probably it's better to use different against Windows "PATH"

Reply:
LD_LIBRARY_PATH and DYLD_LIBRARY_PATH should be send. Also we need add note that it allows dymamic and static linker to find the libirary